src/components/IconButton/IconButton.st.css

```diff
@@ -0,0 +1,55 @@
+:import {
+    -st-from: "wix-ui-core/index.st.css";
+    -st-named: ButtonNext;
+}
+
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback, font, applyOpacity;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultColor: color-5;
+    DisabledColor: color-3;
+}
+
+:vars {
+    /*Overrides*/
+    IconColor: --overridable;
+}
+
+.root {
+    -st-extends: ButtonNext;
+    -st-states: disabled;
+    /*css goes here*/
+    background-color: transparent;
+    text-decoration: none;
+    border: none;
+    display: inline-block;
+    padding: 0;
+    margin: 0;
+    line-height: 0;
+}
+
+.root:hover .icon {
+    opacity: applyOpacity(value(IconColor), 0.6);;
+}
+
+.root:disabled:hover .icon {
+    opacity: 1;
+}
+
+.root:disabled .icon path {
+    stroke: color(value(DisabledColor));
+}
+
+.icon svg {
+    display: block;
+}
+
+.icon path {
+    stroke: color(fallback(value(IconColor), value(DefaultColor)));
+    fill: color(fallback(value(IconColor), value(DefaultColor)));
```

Comment:
I think the `fill` should always be `transparent`, no?

Reply:
no this is not the case, the share icon (from my example has to change fill value)

Reply:
Hmm, but it's not like that in the zeroheight.
Can you send me the spec you're working with?
I'll ask the designers in the meantime.

Reply:
actually I've downloaded the `share` icon from zeroheight

Reply:
and it is designed in the way it has to change `fill` prop

Reply:
I still haven't got an answer from the designers